logging/monolog_exclude_http_codes.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Logging
+   single: Logging; Exclude HTTP Codes
+   single: Monolog; Exclude HTTP Codes
+
+How to Configure Monolog to Exclude Specific HTTP Codes from the Log
+====================================================================
+
+Sometimes your logs become flooded with unwanted HTTP errors, for example,
+403s and 404s. When using a ``fingers_crossed`` handler, you can exclude
+logging these HTTP codes based on the MonologBundle configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
```

Comment:
We need to update the file paths for Symfony 4: `app/config/config.*` -> `config/packages/monolog.*` Thanks!

Reply:
I actually haven't gotten to mess with Symfony 4 much yet, but I just installed `symfony/website-skeleton` to look. It looks like it has:

- `config/packages/dev/monolog.yml`
- `config/packages/prod/monolog.yml`
- `config/packages/test/monolog.yml`

I don't see one just in `config/packages/`. Should I just update the comment to something like:

`# config/packages/{{env}}/monolog.yml`

Reply:
Good catch! Let's add `prod` environment in the example because most of the times you only care about configuring logs in prod (and we use `prod` in most of the examples in the docs). Also, beware that in Symfony 4 the file extension is `.yaml` instead of `.yml`  Thanks!